### ISSUE TITLE
[Suggestion] Do not zero out entries in installed.ini if addon site doesn't respond

### ISSUE THREAD

**Issue by RavenDT** [grrttedwards/wow-addon-updater#187]
Whenever you run the updater, if it the server responds with a 403 for example, that addon entry gets zero'ed out in the `installed.ini` file.

```
>tail addon-updater.log
2021-07-05 13:45:41,080 [DEBUG] [1280] [urllib3.connectionpool] https://www.curseforge.com:443 "GET /wow/addons/zperl HTTP/1.1" 403 None
2021-07-05 13:45:41,080 [DEBUG] [1252] [urllib3.connectionpool] https://www.curseforge.com:443 "GET /wow/addons/world-quest-tracker HTTP/1.1" 403 None
```

```
> dir installed.ini
2021-07-05  13:45                 0 installed.ini
```

This means that the next time the updater is run, if it is able to reach the server, it will download and reinstall all addons, regardless of whether they need to be updated or not.

**Comment by grrttedwards** [comment]
Ah yeah good find. This is a relic of how I decided to speed up the execution from the old original code. I’ll think about how to solve this. Either way it still functions correctly, just causes it to do extra work in this scenario.

**Comment by grrttedwards** [comment]
I misunderstood the root cause of this, and it was trivial to fix. Included in #189 and released via `v.1.9.2`. Thank you!
